org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/dom/Node.java

```diff
@@ -42,7 +42,7 @@
 	/**
 	 * The node is a <code>ProcessingInstruction</code>.
 	 */
-	public static final short PROCESSING_INSTRUCTION_NODE = 7;
+	public static final short PROCESSING_INSTRUCTION_OR_PROLOG_NODE = 7;

```

Comment:
@NikolasKomonen I would like to keep PROCESSING_INSTRUCTION_NODE because I would like to have the same constants than org.w3c.dom.Node.PROCESSING_INSTRUCTION_NODE (perhaps in the future we could implement  org.w3c.dom.Node to benefit with XPath processor, etc). It doesn't exists a PROLOG_NODE constant, that's why I tell me that prolog is a kind of processing instruction. So if you are OK with my idea, please rename to  PROCESSING_INSTRUCTION_NODE.